.github/workflows/tests.yml

```diff
@@ -47,18 +47,12 @@ jobs:
       - name: Package installation
         run: |
           echo "shimming-toolbox installation"
-          pip install .
+          sudo make install
```

Comment:
> When trying to add the launchers to the path, the $HOME differs from the one in the script which seems odd.
> - Echo $HOME in shell : `/root`
> - Echo $HOME in script: `/home/runner`

`/home/runner` should be the correct `$HOME` value for GH Actions. 

I think the reason why the shell script differs is because the shell script is invoked via `sudo make install` (see above), and [`$HOME` is `/root` for the superuser/root account](http://www.linfo.org/slash_root.html).

Reply:
Thanks for the info! I found sudo's `-E` [option](https://www.sudo.ws/man/1.8.13/sudo.man.html#E) which solved my problem by giving superuser privileges but preserves the environment variables of the current user.